apiary.apib

```diff
@@ -614,12 +614,13 @@ These services will manage the individual domain.
 
 # Individuals [/individuals]
 
-### Individual Search [GET /individuals{?name}{?per_page}{?page}]
+### Individual Search [GET /individuals{?name}{?exclude}{?per_page}{?page}]
 
 + Parameters
     + name: Last (string)
     + per_page: 10 (number) - The number of results
     + page: 1 (number) - The page number
+    + exclude: [groupIds]=1,2,3 (string)
```

Comment:
I am wondering if this should have the complex object structure to it.

```
+ exclude (object)
    + group_ids (array[number])
```

I think this would translate to the query string appropriately. Not sure I actually have tried this yet.

``` /individuals?exclude[group_ids]=1,2,3 ```